pg/array_test.go

```diff
@@ -0,0 +1,72 @@
+package pg_test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/elgris/sqrl"
+	"github.com/elgris/sqrl/pg"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestValidArray(t *testing.T) {
+	valid := []struct {
+		op    sqrl.Sqlizer
+		sql   string
+		value string
+	}{
+		{pg.Array([]string{}), "?", "{}"},
+		{pg.Array([]int{}), "?", "{}"},
+		{pg.Array([]float32{}), "?", "{}"},
+		{pg.Array([]float64{}), "?", "{}"},
+		{pg.Array([][]int{}), "?", "{}"},
+		{pg.Array([]string{"foo", "bar", "\"quoted\""}), "?", `{"foo","bar","\"quoted\""}`},
+		{pg.Array([]int{6, 7, 42}), "?", `{6,7,42}`},
```

Comment:
As described above, change from `[]int{6, 7, 42}` to `[]int64{6, 7, 42}` breaks the test

Reply:
Done